common/xrdp_constants.h

```diff
@@ -401,6 +401,7 @@
 #define WAVE_FORMAT_MULAW              0x0007
 #define WAVE_FORMAT_MPEGLAYER3         0x0055
 #define WAVE_FORMAT_OPUS               0x0069
+#define WAVE_FORMAT_AAC_MS             0xA106
```

Comment:
Let's use the same name as RFC or MS document.

https://msdn.microsoft.com/en-us/library/dd341376.aspx
https://tools.ietf.org/html/rfc2361